pkg/providers/instancetype/suite_test.go

```diff
@@ -480,8 +481,10 @@ var _ = Describe("InstanceType Provider", func() {
 				"eviction-hard":           "memory.available<750Mi",
 				"image-gc-high-threshold": "30",
 				"image-gc-low-threshold":  "20",
-				"cpu-cfs-quota":           "true",
+				"cpu-cfs-quota":           "false",
 				"max-pods":                "250",
+				// we don't currently propagate fail-swap-on
```

Comment:
Is this a TODO for this PR to fix?

Reply:
It may be worth waiting for scriptless support in bootstrap to implement this for self-hosted - unless somebody really needs it right now.